tests/vmi_kernel_boot_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"kubevirt.io/kubevirt/tools/vms-generator/utils"
 )
 
-var _ = Describe("[sig-compute]VMI with external kernel boot", func() {
+var _ = FDescribe("[sig-compute]VMI with external kernel boot", func() {
 
```

Comment:
need to remove focus here before we can merge.

Reply:
Thanks for noticing